File: conditionals/time.py
# Jared Lewis, Getting the Time Python
import time

#finrst instance of time in programming
first_time = time.gmtime()
#print(first_time)

#current time in seconds\
current = time.time()
#print(current) #seconds since Jan 1, 1970

#Current date and time like the way we see it
now = time.ctime(current)
#print(now)

#get just a part of the time
local_time = time.localtime(current)
day = local_time.tm_wday
hour = local_time.tm_hour #military time (0-23)
print(hour)

File: conditionals/time.c
// Jared Lewis, Gtting the Time C
#include <stdio.h>
#include <time.h>

int main(void){
    //Time since Jan 1, 1970
    time_t seconds;

    seconds = time(NULL);

    //printf("Seconds since Jan 1, 1970 = %d\n", seconds);

    //Current time
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    printf("Current time & date is %s", asctime(timeinfo));

    //Current hour
    time_t now = time(NULL);

    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour; //military time! (0-23)
    //printf("%d\n", hour);
    
    return 0;
}